examples/main/app.js

```diff
@@ -51,7 +51,6 @@ export default class App extends Component {
         <NotInteractiveExample {...common}/>
         <StyleDiffingExample {...common}/>
         <ClickExample {...common }/>
-        <StyleDiffingExample {...common}/>
```

Comment:
This leaves the style diffing example source file in the tree right? Either delete it, or move it to a separate standalone example.